src/coreclr/jit/gentree.h

```diff
@@ -1770,6 +1770,7 @@ struct GenTree
     inline bool IsVectorZero() const;
     inline bool IsVectorCreate() const;
     inline bool IsVectorAllBitsSet() const;
+    inline bool IsTrueAllMask() const;
```

Comment:
`IsMaskAllBitsSet() const`, for consistency with the naming conventions used elsewhere?